webrtc.html

```diff
@@ -5845,11 +5845,18 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <dd>
               <p>The <code id="dom-rtpreceiver-track">track</code> attribute is
               the track that is associated with this
-              <code><a>RTCRtpReceiver</a></code> object. Note that
-              <code>track.stop()</code> is final, although clones are not
-              affected. Since <code>receiver.track.stop()</code> does not
-              implicitly stop <var>receiver</var>, Receiver Reports continue to
-              be sent.</p>
+              <code><a>RTCRtpReceiver</a></code> object. When a BYE is
+              received on one of the SSRCs in
+              <code><var>receiver</var>.getParameters().encodings[<var>i</var>].ssrc</code>
```

Comment:
This looks right, but I'm wondering if it could be simplified. Would it work to say "When a BYE is received for one of the RTP source streams received by <var>receiver</var>..."?